frontend/src/app/libs/auth-pages/sign-in/sign-in.component.ts

```diff
@@ -0,0 +1,87 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { FormGroup, Validators, FormBuilder, AbstractControl, ValidationErrors } from '@angular/forms';
+import { UserCredentials } from '../../auth/models/user-credentials';
+import { AuthFacadeService } from '../../auth/store/auth/auth.facade';
+import { Router } from '@angular/router';
+import { EntityStatus } from '../../auth/models/entity-status';
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+import { User } from '../../auth/models/user';
+import { IconVisibility } from '../models/icon-visibility';
+import { EntityWrapper } from '../../auth/models/entity-wraper';
+
+@Component({
+	selector: 'app-sign-in',
+	templateUrl: './sign-in.component.html',
+	styleUrls: ['./sign-in.component.scss'],
+})
+export class SignInComponent implements OnInit, OnDestroy {
+	private passwordMinLenth: number = 6;
+	private destroy$: Subject<boolean> = new Subject<boolean>();
+	public signInFormGroup: FormGroup;
+	public submitted: boolean = false;
+	public hidePassword: boolean = true;
+	public userCredential: UserCredentials;
+	public authError: string;
+	public isLoading: boolean = false;
+
+	constructor(private authFacadeService: AuthFacadeService, private formBuilder: FormBuilder, private router: Router) {}
+
+	public ngOnInit(): void {
+		this.authFacadeService.user$.pipe(takeUntil(this.destroy$)).subscribe((user: EntityWrapper<User>) => {
```

Comment:
Could we break .pipe and .subscribe by separated lines?

Reply:
done